lib/dry/validation/hint_compiler.rb

```diff
@@ -61,6 +61,11 @@ def visit_key(node)
         name
       end
 
+      def visit_attr(node)
+        name, _ = node
```

Comment:
That was my intention. Although I wasn't super consistent with that tbh.
Gotta change it in some places
On Sun, 10 Jan 2016 at 18:43, Andy Holland <notifications@github.com> wrote:

> In lib/dry/validation/hint_compiler.rb
> <https://github.com/dryrb/dry-validation/pull/29#discussion_r49278871>:
>
> > @@ -61,6 +61,11 @@ def visit_key(node)
> >          name
> >        end
> >
> > +      def visit_attr(node)
> > +        name, _ = node
>
> Fair point [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dryrb/dry-validation/pull/29/files#r49278871>.
>